src/app/products/products.service.ts

```diff
@@ -19,23 +19,23 @@ export class ProductsService {
   constructor(private http: HttpClient) { }
 
   /**
-   * @param {any} Fetches recurring deposit products.
+   * Fetches recurring deposit products.

```

Comment:
Mention what is being returned in the same line as `@returns` and do not use the word fetches as it is not required but clearly understood from that. See accounting service for reference. As an example:
```
@returns {Observable<any>} Recurring deposit products data.
```